src/MensaWidget/TheWorldMainDockingBar.extension.st

```diff
@@ -0,0 +1,98 @@
+Extension { #name : #TheWorldMainDockingBar }
+
+{
+	#category : #'*MensaWidget-override',
+	#'squeak_changestamp' : 'Louis Christ 6/9/2024 15:10'
+}
+TheWorldMainDockingBar >> mensaMenuOn: aDockingBar [
+	"Create a menu with the options for MensaWidget"
+
+	aDockingBar addItem: [:item |
+		item icon: ((GitAssetLoader for: 'MensaWidget') loadForm: '/icons/mensa_icon.png');
+			addSubMenu: [:menu | 
+				menu addItem: [:menuItem |
+						menuItem contents: 'Potsdam' translated;
+							help: 'open mensa widget' translated;
+							target: self;
+							selector: #openMensaWidget].
+				menu addItem: [:menuItem |
+						menuItem contents: 'Other citys' translated;
+							help: 'open mensa widget location picker' translated;
+							target: self].
+				menu addItem: [:menuItem |
+						menuItem contents: 'Settings' translated;
+							help: 'open mensa widget settings' translated;
+							target: self]]]
```

Comment:
imo, this method would be more readable when the building of the menu items would be extracted. Either something like this:
| mensaIcon potsdamItem otherCitiesItem settingsItem |
    
    mensaIcon := (GitAssetLoader for: 'MensaWidget') loadForm: '/icons/mensa_icon.png'.
    
    potsdamItem := [:menuItem | 
        menuItem contents: 'Potsdam' translated;
            help: 'open mensa widget' translated;
            target: self;
            selector: #openMensaWidget].
    
    otherCitiesItem := [:menuItem | 
        menuItem contents: 'Other cities' translated;
            help: 'open mensa widget location picker' translated;
            target: self].
    
    settingsItem := [:menuItem | 
        menuItem contents: 'Settings' translated;
            help: 'open mensa widget settings' translated;
            target: self].
    
    aDockingBar addItem: [:item |
        item icon: mensaIcon;
            addSubMenu: [:menu |
                menu addItem: potsdamItem.
                menu addItem: otherCitiesItem.
                menu addItem: settingsItem]].
or extract the items into separate methods completely, if that is possible

Reply:
Since it is based on the code they wrote i would still keep it. I know it is our function and we could change it but personally I wouldnt want to go down that rabbithole